### PR TITLE
Feature/social poll module

### PR DESCRIPTION
**Pull request by albertoalaejos**
## Problem
As a CM I want to be able to start a poll

## Solution
Give Content managers the option to create polls so that LU can vote by using an existing module called Poll: https://www.drupal.org/project/poll

## Issue tracker
https://getopensocial.atlassian.net/browse/DS-6940

## How to test
- [ ] Enable the module
- [ ] Login as CM+
- [ ] Click on Content and create a Poll
- [ ] Create a Landing Page
- [ ] Add a Poll Item Section
- [ ] Select the created poll

## Release notes
Content managers now have the option to create polls so that LU can vote.

### CODE REVIEW

File: modules/social_features/social_poll/social_poll.install [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+use Drupal\user\Entity\Role;
+
+/**
+ * Implements hook_install().
+ *
+ * Perform actions related to the installation of social_page.
+ */
+function social_page_install() {
+
```
Comment (ronaldtebrake) [inline modules/social_features/social_poll/social_poll.install]: empty? :D

File: modules/social_features/social_poll/social_poll.module [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+/**
+ * @file
```
Comment (ronaldtebrake) [inline modules/social_features/social_poll/social_poll.module]: Empty file? :D

File: modules/social_features/social_poll/social_poll.install (hunk 1 above)
Comment (albertoalaejos) [reply modules/social_features/social_poll/social_poll.install]: empty file removed ;)

File: modules/social_features/social_poll/social_poll.module (hunk 2 above)
Comment (albertoalaejos) [reply modules/social_features/social_poll/social_poll.module]: empty file removed ;)

File: modules/social_features/social_poll/social_poll.info.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: 'Social Poll'
+description: 'Social Poll'
+type: module
+core: 8.x
+dependencies:
+  - drupal:poll
+  - drupal:block
+  - drupal:field
+  - social:social_landing_page
```
Comment (ronaldtebrake) [inline modules/social_features/social_poll/social_poll.info.yml]: i don't think we want this hard dependency, poll should be able to work without social_landing_pages.

File: modules/social_features/social_poll/src/SocialPollConfigOverride.php [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+   * Load overrides.
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+    $config_factory = \Drupal::service('config.factory');
+
+    $config_names = [
+      'field.field.paragraph.section.field_section_paragraph',
+    ];
+    foreach ($config_names as $config_name) {
```
Comment (ronaldtebrake) [inline modules/social_features/social_poll/src/SocialPollConfigOverride.php]: why a foreach? we only have one config name to check

**Comment by Kingdutch** [comment]
@ronaldtebrake I removed this from the 8.0 milestone

**Pull request closed without merging**